### PR TITLE
add voluntary and fix error.

### PR DESCRIPTION
**Pull request by jessecogollo**
add jessecogollo like voluntary. and fix meetup api events, when meetup return a message without result key.

### CODE REVIEW

File: _harp/posts/_data.json [hunk 1]
```diff
@@ -92,6 +92,14 @@
       "github": "naranjitagolden",
       "avatar": "https://secure.gravatar.com/userimage/80259476/2dad36c56169c60363f29050f78ad24a?size=250",
       "website": "https://twitter.com/naranjitagolden"
+    },
+    "jessecogollo": {
+      "name": "Jesse Cogollo",
+      "bio": "Developer Javascript",
```
Comment (montogeek) [inline _harp/posts/_data.json]: What you think about `JavaScript Developer`?

Comment (jessecogollo) [inline _harp/posts/_data.json]: I don't understand your question.

Comment (RamirezAlex) [inline _harp/posts/_data.json]: @jessecogollo He means it is better to say `JavaScript Developer` instead of `Developer JavaScript`in English.

Comment (jessecogollo) [inline _harp/posts/_data.json]: ahh ok.

**Comment by montogeek** [comment]
dddd

**Pull request merged**